src/main.js

```diff
@@ -70,8 +74,18 @@ categoryBox.addEventListener('click', function (event) {
 });
 
 //functions
+function displayHomePage() {
+  hide(createNewActivityView, 'hidden');
+  show(userActivityInputView, 'hidden');
+}
+
+function displayCreateNewButton() {
+  hide(timerView, 'hidden');
+  show(createNewActivityView, 'hidden');
```

Comment:
Need to add the title switch to  "Completed Activity"

Reply:
The name `hide` or `show` for these functions could be improved. 

If you're using it as a general helper function where you're adding a `rule` parameter, (which could be anything, not just `'hidden'`, but 'disabled' or whatever), maybe a rename to `addClass` or `removeClass` is more appropriate. 

It's definitely a good idea to try to refactor the `hide()`/`show()` and `invisible()`/`visible()` functions, since they're both just using `element.classlist.add/remove()`

I know you've been cautioned about using "toggle", but... these situations is exactly where it's useful.